src/engine/strat_engine/engine.rs

```diff
@@ -213,10 +213,12 @@ impl Engine for StratEngine {
                 } else {
                     let (uuid, pool) = StratPool::initialize(name, blockdev_paths, redundancy)?;
 
-                    let name = Name::new(name.to_owned());
-                    devlinks::pool_added(&name);
-                    self.pools.insert(name, uuid, pool);
-                    Ok(CreateAction::Created(uuid))
+                    let stratis_name = Name::new(name.to_owned());
+                    devlinks::pool_added(&stratis_name);
+                    self.pools.insert(stratis_name, uuid, pool);
+                    let result = CreateAction::Created(uuid);
+                    info!("pool created: uuid {}, name {}", result, name);
+                    Ok(result)
```

Comment:
At this point, this should be changed back to what was here before. We are now doing the logging at the D-Bus layer.

Reply:
Resolved.